modular_zubbers/code/modules/autolathe_components_adv/advpartdisks.dm

```diff
@@ -0,0 +1,146 @@
+//Disks that let autolathes print upgraded parts!
+/obj/item/disk/design_disk/advanced_parts
+	name = "Advanced components autolathe design disk"
+
+/obj/item/disk/design_disk/advanced_parts/Initialize(mapload)
+	. = ..()
+	blueprints += new /datum/design/adv_capacitor
+	blueprints += new /datum/design/adv_scanning
+	blueprints += new /datum/design/nano_servo
+	blueprints += new /datum/design/high_micro_laser
+	blueprints += new /datum/design/adv_matter_bin
+	blueprints += new /datum/design/super_cell
+	blueprints += new /datum/design/super_battery
+	blueprints += new /datum/design/water_recycler
+	blueprints += new /datum/design/card_reader
+
+/datum/design/adv_capacitor
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+/datum/design/adv_scanning
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+/datum/design/nano_servo
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+/datum/design/high_micro_laser
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+/datum/design/adv_matter_bin
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+/datum/design/super_cell
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+/datum/design/water_recycler
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+/datum/design/card_reader
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+
+/obj/item/disk/design_disk/high_tech_parts
+	name = "High-tech components autolathe design disk"
+
+/obj/item/disk/design_disk/high_tech_parts/Initialize(mapload)
+	. = ..()
+	blueprints += new /datum/design/super_capacitor
+	blueprints += new /datum/design/phasic_scanning
+	blueprints += new /datum/design/pico_servo
+	blueprints += new /datum/design/ultra_micro_laser
+	blueprints += new /datum/design/super_matter_bin
+	blueprints += new /datum/design/hyper_cell
+	blueprints += new /datum/design/hyper_battery
+
+/datum/design/super_capacitor
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+/datum/design/phasic_scanning
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+/datum/design/pico_servo
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+/datum/design/ultra_micro_laser
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+/datum/design/super_matter_bin
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+/datum/design/hyper_cell
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+/datum/design/hyper_battery
+	build_type = PROTOLATHE | AWAY_LATHE | AUTOLATHE
+
+/obj/item/disk/design_disk/bluespace_parts
+	name = "Bluespace components autolathe design disk"
+
+/obj/item/disk/design_disk/bluespace_parts/Initialize(mapload)
```

Comment:
Upon further discussion, we've decided that T4 parts should be kept to the protolathes. T3 is fine though.

Reply:
Also fair.